macros/InterfaceData.json

```diff
@@ -1723,6 +1743,10 @@
     "inh"  : "Event",
     "impl" : []
   },
+  "RTCPeerConnectionIceErrorEvent": {
```

Comment:
Yeah, we're currently using a flat event here, which is IIRC a known bug. I think we're generating a flat event but populating it as if it were an RTCPeerConnectionIceErrorEvent. I will double-check.